src/cpp/lpnamer/LinkProfileReader.h

```diff
@@ -10,6 +11,9 @@ class LinkProfileReader {
     LinkProfileReader(){}
 
     static LinkProfile ReadLinkProfile(const std::string& filename);
+    static const std::map<std::string, LinkProfile> getLinkProfileMap(const std::string& capacity_folder, const std::vector<CandidateData>& candidateList);
+private:
+    static void importProfile(std::map<std::string, LinkProfile>& mapLinkProfile, const std::string& capacitySubfolder, const std::string& profile_name);
```

Comment:
Maybe remove `capacitySubfolder `parameter and pass full profile path

Reply:
We can also  put the folder on the constructor (reduce number of parameters).
Make LinkProfile non static.